Scripts/Services/ExploringTheDeep/Items/QuestItems/StorageLocker.cs

```diff
@@ -21,7 +21,7 @@ public class StorageLocker : FillableContainer
 
         private bool m_Active;
         private Parts m_Type;
-        private List<Item> m_Barrels;
+        public List<Item> m_Barrels;
```

Comment:
Standard practice is to make the private field of m_Barrels accessible through a property:

public List<Item> Barrels
{
    get { return m_Barrels; }
    set { m_Barrels = value; }
}